CONTRIBUTING.md

```diff
@@ -26,4 +26,11 @@ by discussion, compromise on the proposed change, or withdrawal of the proposed
 
 # Becoming a Member and Collaborator
 
-TBD
+We are still working to better define the ladder to becoming a full member. The work of User Feedback is primarily non-technical in nature. Given the norms of the Node.js project we still attribute that work through GitHub issues and pull requests.
+
+We would love your contribution and will help you become a collaborator and member.
+Start by joining our bi-weekly meetings. Please [post an issue](https://github.com/nodejs/user-feedback/issues) introducing yourself.
+
+After attending 3 Node.js End User Feedback (NEUF) meetings and becoming a **Contributor**, you will be eligible to become a **Collaborator**.
+
+After 3 months in good standing as a **Contributor**, you will be eligible to become a **Collaborator**.
```

Comment:
I find the last 2 sentences a bit unclear.  Maybe 

After attending 3 Node.js End User Feedback (NEUF) meetings and contributing to the teams efforts, you will be recognized as as **Contributor**.

After 3 months in good standing as a **Contributor**, you will be eligible to become a **Collaborator**.